imagebuildah/build.go

```diff
@@ -222,9 +214,18 @@ type Executor struct {
 	forceRmIntermediateCtrs        bool
 	containerIDs                   []string          // Stores the IDs of the successful intermediate containers used during layer build
 	imageMap                       map[string]string // Used to map images that we create to handle the AS construct.
-	copyFrom                       string            // Used to keep track of the --from flag from COPY and ADD
 	blobDirectory                  string
 	excludes                       []string
+
+	index           int
+	name            string
+	builder         *buildah.Builder
+	preserved       int
+	volumes         imagebuilder.VolumeSet
+	volumeCache     map[string]string
+	volumeCacheInfo map[string]os.FileInfo
+	mountPoint      string
+	copyFrom        string // Used to keep track of the --from flag from COPY and ADD
```

Comment:
Why are you doing this?

Reply:
We were making shallow copies of this structure in `withName()` and then depending on some of the fields to remain shared between them, and some to not be.  It was getting difficult to debug.

Reply:
Can you add a comment about that in the place of the break to explain it, so someone like me does not come back in and merge them back together.

Reply:
Expanded on the comment there.